dashboard/src/api.js

```diff
@@ -84,11 +107,136 @@ export const getAllCities = async () => {
 export const apiClient = axios.create({
   baseURL: apiURL,
   headers: {
-    "accept": "applicaiton/json"
+    "accept": "application/json"
   },
 });
 
 export async function getAllPollutants() {
   const allPollutants = await apiClient.get("/parameters?order_by=name");
   return allPollutants.data;
-}
\ No newline at end of file
+}
+
+
+//Bit of a mess, but I tested it briefly using 'Mexico' as the country and it works that far, at least
+//Helper function getCountryCode returns the 2char code for the country name passed in.
+//Then call the API with the country code and get the data for that country.
+//Then find the measurement for PM2.5 and get the value.
+//Then calculate the AQI value for that PM2.5 value. (Helper functions getBPLimits and getAQILimits are used for this.)
+//Then return the AQI value.
+
+export async function getAQIValue(country) {
```

Comment:
Should we create a separate file for these functions to not muddy our API calls? Maybe in a utils folder under the name AQICalculation.js?